conanfile.py

```diff
@@ -573,12 +582,25 @@ def _getenvpath(var):
 
         if 'glib' in self.deps_cpp_info.deps:
             shutil.move("pcre.pc", "libpcre.pc")
-        with tools.vcvars(self.settings) if self.settings.compiler == "Visual Studio" else tools.no_op():
-            with tools.environment_append({"MAKEFLAGS": "j%d" % tools.cpu_count(), "PKG_CONFIG_PATH": os.getcwd()}):
+
+        build_env = {"MAKEFLAGS": "j%d" % tools.cpu_count(), "PKG_CONFIG_PATH": os.getcwd()}
+        if self.options.qtwebengine:
+            if self.options.python2_bin_dir:
+                build_env['PATH'] = [str(self.options.python2_bin_dir)]
+            if self.settings.compiler in ['gcc', 'clang']:
+                build_env['C_INCLUDE_PATH'] = self.deps_cpp_info['fontconfig'].include_paths
+                build_env['C_PLUS_INCLUDE_PATH'] = self.deps_cpp_info['fontconfig'].include_paths
```

Comment:
it seems to me that you should be using CPLUS_INCLUDE_PATH instead

Reply:
well spot, fixed